bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,178 @@
+# NOTE: This file was automatically generated by docker-entrypoint.sh when you ran docker-compose up
+# If you need to change anything, modify your .env file and restart the container.
+
+# Server encryption keys
+Key.Auth.N = ${kAuthN}
+Key.Auth.K = ${kAuthK}
+Key.Gate.N = ${kGateN}
+Key.Gate.K = ${kGateK}
+Key.Game.N = ${kGameN}
+Key.Game.K = ${kGameK}
+
+# For Python/SDL
+Key.Droid = ${DS_DROIDKEY}
+
+# EXTERNAL server addresses
+File.Host = ${DS_HOST}
+Auth.Host = ${DS_HOST}
+Game.Host = ${DS_HOST}
+
+# Server address/port to bind to...  Leave commented to bind to any
+# address on the default MOULa port (14617)
+Lobby.Addr = 0.0.0.0
+Lobby.Port = 14617
+
+# JSON Server Status over HTTP.  Note:  Disabling the status server will
+# also disable the welcome message below.
+Status.Enabled = true
+Status.Addr = 0.0.0.0
+Status.Port = 8080
+
+# Paths to server data
+File.Root = /opt/dirtsand/lib/filesrv
+Auth.Root = /opt/dirtsand/lib/authsrv
+Sdl.Path = /opt/dirtsand/lib/SDL
+Age.Path = /opt/dirtsand/lib/dat
+
+# Postgres options
+Db.Host = db
+Db.Port = 5432
+Db.Username = ${POSTGRES_USER}
+Db.Password = ${POSTGRES_PASSWORD}
+Db.Database = ${POSTGRES_DB}
+
+# The default Welcome message -- This can be changed while the server
+# is running with the welcome command
+Welcome.Msg = It's ALIVE!
+EOF
+
+# The static_ages.ini file needs to be in the same directory as dirtsand.ini
+if [[ ! -f "/opt/dirtsand/etc/static_ages.ini" ]]; then
+    cp "/opt/dirtsand/static_ages.ini" "/opt/dirtsand/etc/static_ages.ini"
+fi
+
+# Write client ini
+cat << EOF > "${CLIENT_CONFIG_FILE}"
+# This is the server.ini file for plClient. It was automatically generated when you ran docker-compose up
+# You may wish to modify portions of this file in certain situations.
+
+# Server status page shown in the launcher window.
+Server.Status "http://${DS_HOST}:8080/welcome"
+
+# Shard front-end server address.
+Server.Gate.Host "${DS_HOST}"
+
+# Shard name - NOTE: this is currently not visible anywhere.
+Server.DispName "My DIRTSAND Shard"
+
+# URL to a server signup page. DIRTSAND does not provide this by default.
+#Server.Signup "http://example.com/create_account"
+
+# Crypto keys (do not modify)
+Server.Auth.N "${kAuthN}"
+Server.Auth.X "${kAuthX}"
+Server.Game.N "${kGameN}"
+Server.Game.X "${kGameX}"
+Server.Gate.N "${kGateN}"
+Server.Gate.X "${kGateX}"
+EOF
+
+echo -e "\e[32m... Success!\e[0m"
+
+# Ensure the age and SDL files are present.
+# due to mount-point hiding and to ensure a sane environment in case the user broke something...
+echo -e "\e[36mChecking for age and SDL files...\e[0m"
+if [[ ! "$(ls -A /opt/dirtsand/lib/dat)" ]]; then
+    echo -e "\e[33mAge files missing. Copying from base image...\e[0m"
+    cp -R "/opt/dirtsand/lib/moul-scripts/dat/." "/opt/dirtsand/lib/dat"
+fi
+if [[ ! "$(ls -A /opt/dirtsand/lib/SDL)" ]]; then
+    echo -e "\e[33mSDL files missing. Copying from base image...\e[0m"
+    cp -R "/opt/dirtsand/lib/SDL/." "/opt/dirtsand/lib/SDL"
```

Comment:
This should be `cp -R "/opt/dirtsand/lib/moul-scripts/SDL/." "/opt/dirtsand/lib/SDL"`? Otherwise it leaves an empty SDL directory

Reply:
Whoopsie daisy! :trollface: